lua/tabline.lua

```diff
@@ -25,7 +29,7 @@ local tabline = function()
         local tabline_items = {
             icons.get_left_separator(active, bufmodified),                   -- Left separator
             utils.get_item('TabLinePadding', padding, active),               -- Padding
-            icons.get_devicon(active, filename, extension),                  -- DevIcon
+            icons.get_devicon(active, filename, extension, opt.show_icon),   -- DevIcon
```

Comment:
I think we don't need the argument here. We can just get the option with config.get inside the function in stead